anatomy/tracks/concept-exercises.md

```diff
@@ -30,25 +30,31 @@ Concept Exercise metadata is defined in the `exercises.concept` key in the [conf
 
 Each Concept Exercise has its own directory within the track's `exercises/concept` directory. The name of the Concept Exercise directory must match the `slug` property of the Concept Exercise, as defined in the [config.json file](./config-json.md#concept-exercises).
 
-A Concept Exercise is comprised of three types of files:
+A Concept Exercise has three types of files:
 
-- Documentation files: presented to the student to help explain the exercise
-- Metadata files: not presented to the student, but used to define metadata of the exercise
-- Exercise files: the language-specific files, like the implementation and test files
+### Documentation files
 
-Each Concept Exercise directory must contain the following files:
+These files are presented to the student to help explain the exercise.
 
-- `.docs/introduction.md`: introduce the concept(s) that the exercise teaches to the student.
-- `.docs/instructions.md`: provide instructions for the exercise.
-- `.docs/hints.md`: provide hints to a student to help them get themselves unstuck in an exercise.
-- `.meta/config.json`: contains meta information on the exercise.
-- `.meta/design.md`: describe the design of the exercise.
+- `.docs/introduction.md`: introduce the concept(s) that the exercise teaches to the student (required)
+- `.docs/instructions.md`: provide instructions for the exercise (required)
+- `.docs/hints.md`: provide hints to a student to help them get themselves unstuck in an exercise (required)
 
-Besides these files, the following three files must be present, but their file names are track-specific:
+### Metadata files
 
-- Test suite: verify a solution's correctness.
-- Stub implementation: provide a starting point for students.
-- Exemplar implementation: provide an idiomatic implementation that passes all the tests.
+These files are _not_ presented to the student, but used to define metadata of the exercise.
+
+- `.meta/config.json`: contains meta information on the exercise (required)
+- `.meta/design.md`: describe the design of the exercise (required)
+
+### Exercise files
+
+The language-specific files, like the implementation and test files. The names of these files are track-specific.
+
+- Test suite: verify a solution's correctness (required)
+- Stub implementation: provide a starting point for students (required)
+- Exemplar implementation: provide an idiomatic implementation that passes all the tests (required)
```

Comment:
Does the exemplar solution still have a requirement of only using syntax introduced in the exercise or one of its prerequisites?